packages/web/components/Dashboard/LatestContentTabs/Listen.tsx

```diff
@@ -0,0 +1,101 @@
+import { Box, Heading, LoadingState, Text } from '@metafam/ds';
+import React, { useEffect, useState } from 'react';
+import { parse } from 'rss-to-json';
+import useSWR from 'swr';
+
+export const Listen: React.FC = () => {
+  const [items, setItems] = useState<
+    Array<{
+      title: string;
+      src: string;
+      showMore: boolean;
+      description: string;
+    }>
+  >([]);
+
+  const [isLoading, setIsLoading] = useState(true);
+
+  const { data, error } = useSWR(
+    'https://anchor.fm/s/57a641c/podcast/rss',
+    parse,
+  );
+
+  if (error) {
+    console.log(error);
+    setIsLoading(false);
+  }
+
+  useEffect(() => {
+    if (data) {
+      data.items = data.items.map((item) => ({
+        title: item.title,
+        description: item.description.replace(/<\/?[^>]+(>|$)/g, ''),
```

Comment:
tbh i just took a regex i found somewhere that looked liked it successfully stripped html off of the description - didn't put much effort into testing. thx for the suggestion!

Reply:
I later had a change of heart and think you shouldn't be stripping the tags at all, but instead `dangerouslySetInnerHTML`.

¿What's the motivation for removing them?

Reply:
It's because I didn't want to use `dangerouslySetInnerHTML` and just show the description as a block of text. Although it would probably look nicer, this is a potential vulnerability (since any script can be executed in it). So what do you think - do we risk it?

Reply:
Script or style injection are really the only dangers I know of, and the source text will be generated by someone in MetaGame.

That or Anchor goes rogue, which probably isn't going to happen. 😸